recipes/lunasvg/all/patches/2.3.9-0002-rename-inner-selector.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/source/parser.cpp b/source/parser.cpp
+index 714228b..3ade738 100644
+--- a/source/parser.cpp
++++ b/source/parser.cpp
+@@ -1226,8 +1226,8 @@ bool RuleData::matchPseudoClassSelector(const PseudoClassSelector& selector, con
+         return element->parent == nullptr;
+ 
+     if(selector.type == PseudoClassSelector::Type::Is) {
+-        for(auto& selector : selector.subSelectors) {
+-            for(auto& sel : selector) {
++        for(auto& subselector : selector.subSelectors) {
```

Comment:
Hello @toge! Thank you for your pull request. Could you please explain the fix associated to this patch? It seems be a compilation error, probably it should be reported to the upstream as well.

Reply:
@uilianries 
It's a confusing code, but the name of this variable seems to be OK.
It has been already merged at https://github.com/sammycage/lunasvg/pull/147.